android-ktx/src/main/java/com/mirego/trikot/viewModels/ButtonViewModelBinder.kt

```diff
@@ -1,32 +1,32 @@
-package com.mirego.trikot.metaviews
+package com.mirego.trikot.viewModels
```

Comment:
En général je dirais que les nom de package sont au singulier.... 😅

Reply:
As its the library name, it seems to me that it should be plural, like trikot.streams